OracleWebCenterContent/README.MD

```diff
@@ -0,0 +1,16 @@
+# Oracle WebCenter Content on Docker
```

Comment:
The file extension should be lowercase, i.e. `README.md`. Also, please remove all references to "Docker". You should be referring to container images or containers.